virttest/qemu_vm.py

```diff
@@ -1900,6 +1900,15 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        # Add vsock device, cid 0-2 are reserved by system
+        default_cid = 3
+        for vsock in params.objects('vsock'):
+            vsock_params = params.object_params(vsock)
+            guest_cid = vsock_params.get("guest_cid", default_cid)
```

Comment:
Suggest using unified type for it (either int or str, I prefer int)

Reply:
Yep.